### ISSUE TITLE
✅ 🐞 Singular Template's Display Conditions are missing after editing Loop Item templates

### ISSUE THREAD

**Issue by unsubstantiated-Script** [elementor/elementor#24490]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Found that every time we saved updates to the template for a loop item in a loop widget on the front page, somehow the connection between the front page and its template would be broken. We have to go and reset this condition everytime we save the template for a loop item on the front page. 

### Steps to reproduce

Assign a Single Page Template in the theme builder to the Front Page using display conditions. 
Put a loop grid/carousel on the front page. 
Edit the loop item template inline from the editor for either of the above widgets. 
Save it with the inline "Save and Back" or the main "Update" button. 
See that the Single Page template for the Front Page is no longer associated. 
If you directly edit the loop item template from the Edit with Elementor or the Theme Builder (not using the inline editor from a parent template), display conditions are NOT affected and do NOT break. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Darwin
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server (GPL) v5.7.28
	PHP Version: 7.4.30
	PHP Memory Limit: 256M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.1
	Site URL: http://vanillamentor.local
	Home URL: http://vanillamentor.local
	WP Multisite: Yes
	Max Upload Size: 1 MB
	Memory limit: 64M
	Max Memory limit: 256M
	Permalink Structure: /blog/%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.16.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.2
		Author: Elementor.com


== Elements Usage ==
	
	single-page : 1
		container : 3
		heading : 1
		image : 1
		loop-grid : 1
	loop-item : 1
		container : 1
		heading : 1
		image : 1
		theme-post-excerpt : 1


== Settings ==
	
	allow_tracking: yes
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
JS: showing 3 of 3JS: 2023-12-01 22:12:22 [error X 1][../wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.16.4:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-12-01 22:30:37 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.4:3:415598] Cannot read properties of null (reading 'history') 
JS: 2023-12-01 22:30:43 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.4:3:1109666] Cannot read properties of null (reading 'getBoundingClientRect') 

PHP: showing 3 of 3PHP: 2023-12-01 22:19:30 [notice X 8][..c/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-12-01 22:42:25 [notice X 3][..c/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: ..c/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ..c/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ..c/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ..c/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-12-01 22:48:09 [notice X 1][..c/wp-content/plugins/elementor/includes/managers/widgets.php::362] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by rhino-corey** [comment]
+1 We've been experiencing the same issue, but in our case, the archive template conditions get dropped. Using the Save & Back feature from the Archive template is causing our problem. This is a pretty disastrous bug.

Our current work-around is to go directly to the loop item template for editing and not use the Save & Back feature.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.23.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
